### ISSUE TITLE
Backlog Ideas

### ISSUE THREAD

**Issue by senavi888** [n4ze3m/dialoqbase#227]
Hello! I would like to give ideas for product development.

1. Adding to the knowledge base not only a separate page of the site, but the entire site. Parse an entire website, limiting it to a domain. It is necessary to provide the ability to add a list of exclusions of pages that should not be processed.
2. Adding doxc tables to the database, for example
3. Adding a link to the group and channels from telegrams to the database

I also noticed that when I add a video from YouTube, my database is not filled, and the video is being processed. The mistake may be that it does not recognize different languages, but only works for English speech.
As an option, you can use subtitles that YouTube itself automatically creates for the video, so as not to waste server resources.

I will be glad if my thoughts help you.

**Comment by n4ze3m** [comment]
Hey, thank you for your suggestions! I will look into them; most things are possible, except for a few.

I really like the fourth suggestion. I think I will add a button to the bot settings page called 'Create A Copy' or something similar.

I will implement a few of the suggested updates in the next update, hopefully this weekend or the next

**Comment by senavi888** [comment]
Yes, one more feature that would be useful. This is when adding, for example, the URL of a site or page, to automatically request an update, for example, once a day, this is useful when updates on the page are checked and the database is updated. This can be useful when updating the class schedule, order status, etc.

**Comment by n4ze3m** [comment]
Hey, thank you for sponsoring me. I will add requested features in the coming updates :) If you have any other suggestions, feel free to share

**Comment by senavi888** [comment]
You're welcome! :) Thank you for your hard work.

**Comment by n4ze3m** [comment]
Hey, a new version has been released which supports a 'make a copy of bot' button. Currently, it creates a copy of the existing bot, excluding integration settings due to restrictions in Telegram, WhatsApp, and Discord, which do not allow two listeners at a time.

![image](https://github.com/n4ze3m/dialoqbase/assets/39720973/c2bd70df-e9ae-487e-948d-906d27148abb)

**Comment by n4ze3m** [comment]
Wait for a few hours; the latest version is still building. If you are using the official Dialoqbase Docker image, otherwise, you can pull it now

**Comment by n4ze3m** [comment]
Hey, v1.7.6 has been released, and YouTube transcripts are now supported. However, there is a drawback: it won't work with age-restricted videos for now. We will fix this in upcoming updates

![image](https://github.com/n4ze3m/dialoqbase/assets/39720973/9046267b-fac6-4b19-b9b8-f30e62710396)

**Comment by senavi888** [comment]
You're a release machine! 🔥❤️ thanks so much!

**Comment by n4ze3m** [comment]
I'm adding a few backlog idea features to Dialogbase this week. You mean a separate knowledge management system that can easily link and unlink with the chatbot, right?

**Comment by senavi888** [comment]
> I'm adding a few backlog idea features to Dialogbase this week. You mean a separate knowledge management system that can easily link and unlink with the chatbot, right?

Hi! If you're talking about point 3.
I meant parsing a group or a channel.

For example, if it is a public channel, then give a link to it to the system, the system will follow the link, unload all the history and upload it to the knowledge base.

For example, if it is a group, then you need to add a bot to the group, and the bot will parsing all the messages from this group.

It would be possible to make parsing updates on a schedule.
